docs/tutorial.md

```
@@ -0,0 +1,88 @@
+
+## Basic Usage
+
+Create `app.py` with the content below:
+
+```python
+--8<-- "./examples/tutorial/01_basic_usage.py"
+```
+
+Unlike other Streamlit components, `webrtc_streamer()` requires the `key` argument as a unique identifier. Set a unique, descriptive string as its value to identify each component instance.
+
+Then run it with Streamlit and open http://localhost:8501/:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)